GsaGHTests/Helpers/GH/Legend/LegendMenuBuilderTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Windows.Forms;
+
+using GsaGH.Components;
+using GsaGH.Helpers;
+
+using Moq;
+
+using Xunit;
+
+namespace GsaGHTests.Helpers {
+  [Collection("GrasshopperFixture collection")]
+  public class LegendMenuBuilderTests {
+    private Mock<Contour1dResults> _componentMock;
+    private double _initialScale;
+    private ToolStripMenuItem _menuItem;
+
+    public LegendMenuBuilderTests() {
+      _componentMock = new Mock<Contour1dResults>(); // any component you want
+      var updateUiMock = new Mock<Action>();
```

Comment:
do we need to mock the update? I mean you will remove the UpdateUi either way but this one could have been: `() => {}`